ubyssey/helpers.py

```diff
@@ -7,6 +8,8 @@
 from django.conf import settings
 from django.http import Http404
 from django.db import connection
+import django.db.models
```

Comment:
This line imports all definitions from `django.db.models`, but the line below imports specific members. I would remove this line in favour of the one below.